src/test/CMakeLists.txt

```diff
@@ -772,7 +772,7 @@ target_link_libraries(unittest_libcephfs_config cephfs)
 # unittest_rbd_replay
 add_executable(unittest_rbd_replay
   test_rbd_replay.cc
-  $<TARGET_OBJECTS:parse_secret_objs>
+#  $<TARGET_OBJECTS:parse_secret_objs>
```

Comment:
On 30-6-2016 12:05, Kefu Chai wrote:
> In src/test/CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10024#discussion_r69106054>:
> 
>> @@ -772,7 +772,7 @@ target_link_libraries(unittest_libcephfs_config cephfs)
>>  # unittest_rbd_replay
>>  add_executable(unittest_rbd_replay
>>    test_rbd_replay.cc
>> -  $<TARGET_OBJECTS:parse_secret_objs>
>> +#  $<TARGET_OBJECTS:parse_secret_objs>
> 

Sloppy exclude. Sorry.
parse_secrect_objs used KVS and that is not on FreeBSD.
Although it seems like a nice feature